sass/application.scss

```diff
@@ -214,3 +214,9 @@ $govuk-page-width: 1170px;
 .case-notes-details {
   flex-basis: 80%;
 }
+
+pre {
+  white-space: pre-wrap;       /* css-3 */
+  white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
+  word-wrap: break-word;       /* Internet Explorer 5.5+ */
+ }
```

Comment:
`pre` elements are very permissive so unless you tell them to, they do not constrain content at all.